python/src/puml/gen_puml.aac

```diff
@@ -0,0 +1,103 @@
+plugin:
+  name: Generate PlantUML Diagrams
+  package: aac.plugins.puml
+  description: |
+    An AaC plugin that enables PlantUML diagrams to be generated from AaC
+    definitions.
+  commands:
+    - name: puml-component
+      help_text: Converts an AaC-defined system to a PlantUML component diagram.
+      input:
+        - name: architecture-file
+          type: file
+          description: |
+            A path to a YAML file containing an AaC-defined system from which to
+            generate a PlantUML component diagram.
+        - name: output-directory
+          type: directory
+          description: |
+            The output directory into which the PlantUML (.puml) diagram file
+            will be written.
+      acceptance:
+        name: Component Diagram Test
+        scenarios:
+          - name: Output PlantUML component diagram for specified .aac file and output path.
```

Comment:
It won't always be a `*.aac` file, it can be a `*.yml` as well. So, should not limit it this way, or need to put a similar test case in for a `*.yml` file. 

```suggestion
          - name: Output PlantUML component diagram for specified file and output path.
```